Release 4.0.2 (#27)
blackbaud/skyux-data-manager: CHANGELOG.md

# 4.0.2 (2021-01-28)

> - Fixed the data manager toolbar component to only excute searches that are applied via pressing "Enter" or clicking the search button. [#25](https://github.com/blackbaud/skyux-data-manager/pull/25)

Reviewer: ```suggestion
- Fixed the data manager toolbar component to only execute searches when users press "Enter" or select the search button. [#25](https://github.com/blackbaud/skyux-data-manager/pull/25)
```
